From ec95c016232f63827c8eebcee8e0e8ba3a470981 Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Wed, 22 Apr 2020 17:20:49 +0800
Subject: [PATCH] arm64: rockchip_defconfig: Disable kernel logo

If you want to enable kernel logo set the following state:
  1. enable CONFIG_FRAMEBUFFER_CONSOLE
  2. enable CONFIG_LOGO
  3. disable route_HDMI/eDP/MIPI state at dts file;

Change-Id: Ibe35ef6e296718cde9871c8cc2e61e1621c4d6a6
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 304fb226b648..95129636fc84 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -596,9 +596,6 @@ CONFIG_ROCKCHIP_MPP_VDPU1=y
 CONFIG_ROCKCHIP_MPP_VEPU1=y
 CONFIG_ROCKCHIP_MPP_VDPU2=y
 CONFIG_ROCKCHIP_MPP_VEPU2=y
-CONFIG_LOGO=y
-# CONFIG_LOGO_LINUX_MONO is not set
-# CONFIG_LOGO_LINUX_VGA16 is not set
 CONFIG_SOUND=y
 CONFIG_SND=y
 CONFIG_SND_DYNAMIC_MINORS=y
-- 
2.35.3

